src/boot/idle.c

```diff
@@ -1,3 +1,5 @@
+#include "stdbool.h"
```

Comment:
Can you move this below the `prevent_bss_reordering`? We try to keep `prevent` as the first so it is easier to find and remove as needed.